src/SDKs/DataMigration/DataMigration.Tests/DataMigration.Tests.csproj

```diff
@@ -13,8 +13,6 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.3.2" />
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
-    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```

Comment:
@hitenjava Does the project build after removing the two lines? Did you remove them using the Nuget Package Manager? There may still be references to the binaries in the csproj.